RELEASE_NOTES.txt

```diff
@@ -1,6 +1,177 @@
+* Fixed warnings about closed socket.
+* Fixed wrong old index size in the defragmentation procedure.
+* Forbid a non-transactional IgniteCache#clear() inside a transaction.
+* Implemented custom user defined metrics.
+* Implemented transaction aware SQL and Scan queries.
+* Improved check of previous node on new incoming server connection.
+* Moved Ignite sources to Java 11.
+* Optimized UUID serialization for communication messages.
+* Optimized entries expiration for persistent caches.
+* Removed deprecated TcpCommunicationSpi shared memory port configuration.
+* Removed deprecated authorization methods from Security Context.
+* Updated Jetty dependency version to 9.4.55.
+* Updated Netty dependency version to 4.1.101.Final.
+* Updated PostgreSQL JDBC dependency version to 42.7.3.
+* Updated Spring dependency version to 5.3.39.
+* Updated commons-io dependency version to 2.17.0.
+* Updated grpc-context version to 1.62.2.
+* Updated janino dependency version to 3.1.12.
+* Updated jetty dependency version to 9.4.56.v20240826.
+* Updated json-path dependency version to 2.9.0.
+* Updated jtidy dependency version to 1.0.5.
+* Updated log4j2 dependency version to 2.22.0.
+* Updated opencensus dependency version to 0.31.1.
+* Updated snappy dependency version to 1.1.10.7.
+* Updated zookeeper dependency version to 3.8.4.
+
+SQL:
+* Added ability to create user defined SQL views.
+* Added long queries tracking for DML operations.
+* Fixed AssertionError due to cache size overflow.
+* Fixed absence of warnings for long running queries with lazy flag.
+* Fixed assertion error on extra column in H2 ResultInterface#currentRow.
+* Fixed nested subquery optimization.
+* Fixed table statistics periodical update.
+
+SQL Calcite:
+* Added ability to extend SQL engine with new system functions/operators.
+* Added arithmetic overflow handling.
+* Added date formatting/parsing functions with custom format.
+* Added hint for join type.
+* Added memory quotas control for Cursor.getAll() method.
+* Added operations authorization.
+* Added support for SQL hints at tables.
+* Added support for bitwise aggregates (BIT_AND, BIT_OR, BIT_XOR).
+* Added support for bitwise operations (BITAND, BITOR, BITXOR).
+* Added support of OR operator with dynamic parameters for index scans.
+* Allowed grouping by alias and ordinal value.
+* Fixed 'typeof' function behavior when operand throws an error (it fails now).
+* Fixed NVL/DECODE failure when used with parameters of different data types.
+* Fixed assertion error for multiline statements in thin client.
+* Fixed binary object dynamic parameter processing.
+* Fixed column nullability check for data types with precision or scale.
+* Fixed compiled expression cache key for correlated variable (to avoid cache collisions for correlated variables with different data types).
+* Fixed complex key index scans (binary objects comparison).
+* Fixed correlated nested loop LEFT join failure.
```

Comment:
Should there be back quotes ` here and below?

`LEFT`, `TIMESTAMP`, `COUNT(DISTINCT)`